saw-core/src/SAWCore/Prim.hs

```diff
@@ -88,24 +88,20 @@ append :: Int -> Int -> t -> Vec t e -> Vec t e -> Vec t e
 append _ _ _ (Vec t xv) (Vec _ yv) = Vec t ((V.++) xv yv)
 
 -- at :: (n :: Nat) -> (a :: sort 0) -> Vec n a -> Nat -> a;
-at :: Int -> t -> Vec t e -> Int -> e
-at _ _ (Vec _ v) i = v ! i
+at :: () -> () -> Vec t e -> Int -> Maybe e
+at _ _ (Vec _ v) i = v V.!? i
```

Comment:
I'm not familiar with this convention of passing explicit `()` arguments to SAWCore-related functions like this. What purpose do the `()`s serve? Can they be removed?